src/bridge/graphs/peg_out.rs

```diff
@@ -244,13 +253,33 @@ impl PegOutGraph {
             &context.operator_taproot_public_key,
             &context.n_of_n_taproot_public_key,
         );
+        let (connector_6, connector_6_winternitz_secret) =
```

Comment:
[MINOR] The constructor still returns a collection of secrets, so I suggest to make this var name plural. Currently there's only one secret in connector 6, but if that changes in the future we won't have to remember to change this name.
```suggestion
        let (connector_6, connector_6_winternitz_secrets) =
```